src/main/java/com/uid2/admin/salt/SaltRotation.java

```diff
@@ -246,6 +247,22 @@ private void logSaltAges(String saltCountType, TargetDate targetDate, Collection
         }
     }
 
+    private void logBucketFormatCount(TargetDate targetDate, SaltEntry[] preRotationSalts, SaltEntry[] postRotationSalts) {
+        int newKeyBucketCounter = 0, totalKeyBucketCounter = 0, totalSaltBucketCounter = 0;
```

Comment:
I think this deserves a comment - mention the migration, that the salts are the old format and we're transitioning to encryption keys. Mention that this is to monitor the migration.